onnx/test/reference_evaluator_test.py

```diff
@@ -1965,6 +1965,96 @@ def test_stft_with_window(self):
         got1 = ref1.run(None, feeds)
         assert_allclose(expected, got1[0])
 
+    def test_split(self):
```

Comment:
I wonder if we should just rely on ONNX node test cases? I believe the reference implementation is tested against ONNX node test cases right? Why duplicate test-cases?

Reply:
The test cases do not cover all the possible cases, negative values for axis, all possible options for a parameter... There is no test case for ai.onnx.ml.